### PR TITLE
Bug-8814 Paragraphs Welsh language toggle issue ( + General Translation bug fixes - assignment headings) 

### PR DESCRIPTION
**Pull request by peter-mal**
Extends Language Toggle functionality to store list of localeResources for each page as we navigate through a claim to try to ensure that all required bundles are loaded and used when toggling language

Updates Assignment and Default form component to use the Pconnect localization call rather than PCore localization call as this appears to respond 'better' to locale resource loading (avoiding issue we have sometimes seen where localeStore was not updated/populated at time of PCore localisation call.
This has allowed a simplified Assignment Heading logic, simply calling localisation inline.

Finally, have also updated error message localisation in Assignment component to fix issue with missing localeReference due to new localisation call.

Also removes locale resource load calls from throughout the app as everything should be loaded correctly via LanguageToggle component.

### CODE REVIEW

File: src/components/override-sdk/infra/View/View.tsx [hunk 1]
```diff
@@ -35,8 +35,8 @@ export default function View(props) {
   useEffect(() => {
     PCore.getPubSubUtils().subscribe('languageToggleTriggered', locale => {
       if (!locale.localeRef.includes(props.localeReference)) {
-        locale.localeRef.push(props.localeReference);
-        PCore.getLocaleUtils().loadLocaleResources([props.localeReference]);
+        // locale.localeRef.push(props.localeReference);
```
Comment (Carl-M-H) [inline src/components/override-sdk/infra/View/View.tsx:38]: This section of code 38 - 39 is commented out, is this needed if not can we take it out?

File: src/samples/ChildBenefitsClaim/index.tsx [hunk 2]
```diff
@@ -464,11 +464,11 @@ export default function ChildBenefitsClaim() {
 
       // TODO : Consider refactoring 'en_GB' reference as this may need to be set elsewhere
```
Comment (Carl-M-H) [inline src/samples/ChildBenefitsClaim/index.tsx:465]: Is this ToDo action still needed if so has it been recorded on Agile Studio?

File: src/samples/ChildBenefitsClaim/index.tsx [hunk 3]
```diff
@@ -464,11 +464,11 @@ export default function ChildBenefitsClaim() {
...
       PCore.getEnvironmentInfo().setLocale(sessionStorage.getItem('rsdk_locale') || 'en_GB');
-      PCore.getLocaleUtils().resetLocaleStore();
-      PCore.getLocaleUtils().loadLocaleResources([
+      // PCore.getLocaleUtils().resetLocaleStore();
+      /* PCore.getLocaleUtils().loadLocaleResources([
```
Comment (Carl-M-H) [inline src/samples/ChildBenefitsClaim/index.tsx:468]: This section of code 467 - 471 has been commented out, is this still needed?

File: src/samples/EducationStart/reuseables/PegaSetup.tsx [hunk 4]
```diff
@@ -314,11 +314,11 @@ export function startMashup(
...
-    PCore.getLocaleUtils().loadLocaleResources([
+    // PCore.getLocaleUtils().resetLocaleStore();
+    /* PCore.getLocaleUtils().loadLocaleResources([
       PCore.getLocaleUtils().GENERIC_BUNDLE_KEY,
       '@BASECLASS!DATAPAGE!D_LISTREFERENCEDATABYTYPE'
-    ]);
+    ]); */
     initialRender(renderObj, setAssignmentPConnect, _AppContextValues);
 
     // PM!! operatorId = PCore.getEnvironmentInfo().getOperatorIdentifier();
```
Comment (Carl-M-H) [inline src/samples/EducationStart/reuseables/PegaSetup.tsx:324]: This has been commented out is this still needed?

File: src/samples/UnAuthChildBenefitsClaim/index.tsx [hunk 5]
```diff
@@ -363,11 +363,11 @@ export default function UnAuthChildBenefitsClaim() {
 
       // TODO : Consider refactoring 'en_GB' reference as this may need to be set elsewhere
       PCore.getEnvironmentInfo().setLocale(sessionStorage.getItem('rsdk_locale') || 'en_GB');
-      PCore.getLocaleUtils().resetLocaleStore();
+      /* PCore.getLocaleUtils().resetLocaleStore();
```
Comment (Carl-M-H) [inline src/samples/UnAuthChildBenefitsClaim/index.tsx:366]: This section of code 366 - 370 has been commented out is this still needed?

File: src/samples/ChildBenefitsClaim/index.tsx (hunk 2 above)
Comment (peter-mal) [reply src/samples/ChildBenefitsClaim/index.tsx:465]: Will remove the Todo action.

While i'm sure there may be some value in refactoring this, it's currently not clear that this is going to be a constant across all projects, so will look into this if the logic is moved into a shared component.

**Pull request merged**